components/ai_chat/core/browser/conversation_driver.h

```diff
@@ -69,9 +78,7 @@ class ConversationDriver {
   void OnConversationActiveChanged(bool is_conversation_active);
   void AddToConversationHistory(mojom::ConversationTurn turn);
   void UpdateOrCreateLastAssistantEntry(std::string text);
-  void MakeAPIRequestWithConversationHistoryUpdate(
```

Comment:
FYI this is still called in brave/ios/browser/api/ai_chat/ai_chat.mm, didn't check others, you might want to take a pass of this file and update accordingly.

Reply:
Yes CI is failing on iOS because I didn't yet update the subclass. I'm fixing that now.